usmqe/api/tendrlapi/common.py

```diff
@@ -319,16 +319,19 @@ def create_cluster(
         self.check_response(response, asserts_in)
         return response.json()
 
-    def import_cluster(self, cluster_id, enable_profiling="yes", asserts_in=None):
+    def import_cluster(
+            self, cluster_id, profiling="enable", short_name=None,
```

Comment:
I have just two questions:
* what if `short_name` is set to `None` - does this mean, that the Cluster name is not configured? (just wan't to be sure, that it is not somewhere converted to string `"None"`.
* and second question - can I set the third option for the `profiling` parameter - "do not change the current state"?

Reply:
* In Python `requests` module is None translated to json value `null`.
 * Good catch, there is also *leave-as-is* value for the api parameter. I updated the code so that if `profiling` is not set then it is set to *leave-as-is*.